From 6a522a37f57ee18c0544ec4560a9cb1065887f80 Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@nxp.com>
Date: Fri, 13 Jul 2018 10:39:21 +0800
Subject: [PATCH 4260/5242] MLK-18605-7 drm/bridge: sec-dsim: correct a
 register macro misspelt

commit  f1301d3895ec99145b1325971c39acb5eac247a6 from
https://source.codeaurora.org/external/imx/linux-imx.git

The macro 'CONFIG_NON_CONTINUOUS_CLOCK_LANE' has been misspelt
to 'CONFIG_NON_CONTINOUS_CLOCK_LANE'. So correct it.

Signed-off-by: Fancy Fang <chen.fang@nxp.com>
(cherry picked from commit 24a05825b7299fc2f60768468fafbbc9a33804ad)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/gpu/drm/bridge/sec-dsim.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/bridge/sec-dsim.c b/drivers/gpu/drm/bridge/sec-dsim.c
index 695bbde..a1575ee 100644
--- a/drivers/gpu/drm/bridge/sec-dsim.c
+++ b/drivers/gpu/drm/bridge/sec-dsim.c
@@ -98,7 +98,7 @@
 #define TIMEOUT_SET_BTAOUT(x)		REG_PUT(x, 23, 16)
 #define TIMEOUT_SET_LPDRTOUT(x)		REG_PUT(x, 15,  0)
 
-#define CONFIG_NON_CONTINOUS_CLOCK_LANE	BIT(31)
+#define CONFIG_NON_CONTINUOUS_CLOCK_LANE	BIT(31)
 #define CONFIG_CLKLANE_STOP_START	BIT(30)
 #define CONFIG_MFLUSH_VS		BIT(29)
 #define CONFIG_EOT_R03			BIT(28)
-- 
1.7.9.5

